edtrio-client/src/App.css

```diff
@@ -33,3 +33,13 @@
   width: 100%;
   height: 100%;
 }
+
+html,
+body {
+  height: 100%;
+}
+
+span {
```

Comment:
can we turn this into styled components as well? I think it'd be cleaner

Reply:
Well, we intended to change every occurrence of `span`s in the app, including those introduced by slate.js. In the long term, it would be nice to drop the whole `App.css` and completely rely on styled components. For now, I'm not sure how to change that. However, I fear, this might interfere with some styling, such as the line break around inline code elements.

Reply:
alright. Agree, more of a nice-to-have, def not a must. 👍